### ISSUE TITLE
Incorrect scrollbar calculation: plus should be replaced by minus

### ISSUE THREAD

**Issue by truedim** [ocornut/imgui#1574]
https://github.com/ocornut/imgui/blob/486f0a8ba0eda020d912b2058f13c30e74a77aa2/imgui.cpp#L4675

**Comment by ocornut** [comment]
Thank you, it looks like a bug introduced in c36e586cce1279b419ab4b443ca6ce63b03ce7cc (I replaced `+ ItemSpacing.y - ScrollbarSize` by `+ ScrollbarSize`. Fixed now.

It's easy to confirm by resizing this window manually and comparing how the ScrollbarY appears depending on whether ScrollbarX is enabled.
```
ImGui::Begin("Test #1574", NULL, ImGuiWindowFlags_HorizontalScrollbar);
ImGui::Button("##xxx", ImVec2(500,500));
ImGui::End();
```

Thank you!

**Comment by ocornut** [comment]
Very surprisingly, this got accidentally broken back in a0994d74c in 2019 (to a lesser extent than the original report).
Fixed again with 6cefd4f.
Added a test https://github.com/ocornut/imgui_test_engine/commit/45c1fbbf359bd96b621293cfe05e399e9d9cc2a1
